### ISSUE TITLE
[iOS 13.3] WebView: cant switch to webView (issue returned)

### ISSUE THREAD

**Issue by amedvedjev** [appium/appium#13906]
## The problem

Appium 1.16.0 (from 3 Feb 2020) cant switch into WebView.
In logs we see that RemoteDebugger nicely sees all webViews but responding only Native.
Same with Safari - no problem to see webViews.

## Environment

* Appium version (or git revision) that exhibits the issue: Appium 1.16.0 (all commits up to 3 Feb 2020)
* Last Appium version that did not exhibit the issue (if applicable): Appium 1.16.0 (2 weeks before)
* Desktop OS/version used to run Appium: MacOS
* Mobile platform/version under test: iOS
* Real device or emulator/simulator: Simulator

## Link to Appium logs
<details>
  <summary>Appium package</summary>

  ```
{
  '@babel/runtime': '^7.6.0',
  'appium-android-driver': '^4.21.0',
  'appium-base-driver': '^5.0.0',
  'appium-espresso-driver': '^1.18.0',
  'appium-fake-driver': '^0.x',
  'appium-flutter-driver': '^0',
  'appium-ios-driver': '4.x',
  'appium-mac-driver': '1.x',
  'appium-support': '2.x',
  'appium-tizen-driver': '^1.1.1-beta.4',
  'appium-uiautomator2-driver': '^1.40.0',
  'appium-windows-driver': '1.x',
  'appium-xcuitest-driver': '^3.8.0',
  'appium-youiengine-driver': '^1.2.0',
  argparse: '^1.0.10',
  'async-lock': '^1.0.0',
  asyncbox: '2.x',
  bluebird: '3.x',
  'continuation-local-storage': '3.x',
  dateformat: '^3.0.3',
  'find-root': '^1.1.0',
  lodash: '^4.17.11',
  longjohn: '^0.2.12',
  npmlog: '4.x',
  request: '^2.81.0',
  'request-promise': '4.x',
  semver: '^6.1.2',
  'source-map-support': '0.x',
  teen_process: '1.x',
  winston: '3.x',
  'word-wrap': '^1.2.3',
  fsevents: '2.x'
}
  ```
</details>

Logs [link](https://gist.github.com/amedvedjev/e9093ee0d54b75c7e23c002d1f3fff65)

**Comment by imurchie** [comment]
> Same with Safari - no problem to see webViews.

I'm not sure what this means. Are webviews reported or not?

**Comment by amedvedjev** [comment]
@imurchie It means if i run Safari browser on macbook i can see webViews in developer menu with connected phone and running app under test.

**Comment by imurchie** [comment]
Ah. Ok. Thanks. I made some changes to the internals of the remote debugger, and saw no problems. I will look into this now.

**Comment by amedvedjev** [comment]
i see "appium-remote-debugger@^8.0.0" i use now.

**Comment by imurchie** [comment]
Yes, that is the latest.

**Comment by imurchie** [comment]
@amedvedjev Can you try updating, to get `appium-remote-debugger` version `8.0.1`?

**Comment by amedvedjev** [comment]
@imurchie nice. fixed!

**Comment by imurchie** [comment]
Excellent! Glad it worked, and I hope the changes that broke things will let us do some more advanced work on MobileSafari!

**Comment by amedvedjev** [comment]
@imsardine sorry. i checked on Simulator but did not check on real device. it does not work with real device...
[log](https://gist.github.com/amedvedjev/90ddd6ec4d97623d0e4e77f5672bab80)

**Comment by imurchie** [comment]
Real device support ought to be fixed in `8.1.0`. I came to the same position yesterday.

**Comment by amedvedjev** [comment]
thanks. checked! working.
